src/actions/index.js

```diff
@@ -1,25 +1,69 @@
 // actions call reducers
+import fetch from 'isomorphic-fetch';
 
-let nextId = 0
+let nextId = 0;
 
-// calling reducers
-export const addTodo = (text) => ({
-    type: 'ADD_TODO',
+export const ADD_TODO = 'ADD_TODO';
```

Comment:
Export all these constants inside one object in order to make it more reusable across other files.